.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,43 @@
+name: CI-CD
+on: [push, pull_request]
+jobs:
+  test-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      # build
+      - uses: actions/checkout@v3
+      - run: >
+          mv django.env.sample django.env &&
+          mv postgres.env.sample postgres.env &&
+          mv mbtiles.env.sample mbtiles.env &&
+          touch auth.env
+      - env:
+          SERVICE_KEY_BASE64: ${{ secrets.SERVICE_KEY_BASE64 }}
+        run: echo "${SERVICE_KEY_BASE64// /}" | base64 --decode > config/firebase.json
+      - run: make pull-images
+      - run: make system-image # TODO: just for testing
```

Comment:
I would propose to extract make and push jobs for system-image to a separate workflow, which could be triggered manually.

https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

Reply:
Good idea!